crates/nu-command/src/debug/experimental_options.rs

```diff
@@ -18,6 +18,8 @@ impl Command for DebugExperimentalOptions {
                     (String::from("enabled"), Type::Bool),
                     (String::from("status"), Type::String),
                     (String::from("description"), Type::String),
+                    (String::from("since"), Type::String),
+                    (String::from("pr"), Type::String),
```

Comment:
shouldn't this be issue?
```suggestion
                    (String::from("issue"), Type::String),
```